### ISSUE TITLE
Unable to login to GitHub account

### ISSUE THREAD

**Issue by Farmils** [ethereum/remix-project#5405]
**Reproduce**
1. Go to the git tab

2. Click github setup

3. Click on login with github

4. After a few seconds, an error will appear in the developer console: Axios error

5. View the error

**Expected behavior**
When you click on login with github, you are expected to be redirected to the github page for authorization.

**Screenshots**
![image](https://github.com/user-attachments/assets/25e851fc-f77f-4aa2-9dfe-63bc9668644d)

**Desktop:**
 - OS: Windows10
 - Browser chrome
 - Version 130.0.6723.119

**Comment by bunsenstraat** [comment]
<img width="409" alt="Screenshot 2024-11-22 at 21 32 27" src="https://github.com/user-attachments/assets/6a0c11d6-4c24-4c19-8a9f-3ae82a228157">

do you see a screen with a code like this when you click login with github?

**Comment by Farmils** [comment]
No, I press the button, nothing happens, an error appears in the screenshot above

**Comment by Farmils** [comment]
> <img width="409" alt="Screenshot 2024-11-22 at 21 32 27" src="https://github.com/user-attachments/assets/6a0c11d6-4c24-4c19-8a9f-3ae82a228157">
> 
> do you see a screen with a code like this when you click login with github?

No, I press the button, nothing happens, an error appears in the screenshot above
